controllers/articles.controller.js

```diff
@@ -1,14 +1,32 @@
-const {selectArticle} = require('../models/articles.model')
+const { response } = require('../app');
+const {selectArticle,
+    allArticlesData} = require('../models/articles.model')
 
 function getArticleById(request, response, next) {
     const {article_id} = request.params;
     selectArticle(article_id)
     .then((articleData) => {
         response.status(200).send({article: articleData})
-    })
+    }) 
     .catch((err) => {
         next(err)
     })
 }
 
-module.exports = {getArticleById}
\ No newline at end of file
+function getAllArticlesData(request, response, next) {
+    allArticlesData()
+    .then((allArticlesData) => {
+        const formattedArticlesData = allArticlesData.map(article => {
+            const copyArticle = {...article};
+            copyArticle.votes = +copyArticle.votes;
+            copyArticle.comment_count = +copyArticle.comment_count;
+            return copyArticle
+        })
```

Comment:
There is a way to get SQL to return the `comment_count` as an _INT_ so that you don't have to go through the process of mapping the data, I'll put it in a comment on the model